### ISSUE TITLE
GA

### ISSUE THREAD

**Issue by shrayasr** [PySangamam/website#19]
Introduce Google Analytics to get a sense of what's going on

Caveat: We're falling into the realm of "tracking" people. Are we OK to do this? 

cc @zerothabhishek @bravegnu

**Comment by bravegnu** [comment]
I think it is a good thing to have. It is only at the macro level. It is useful, for things like understanding the effectiveness of promotions, etc.

**Comment by shrayasr** [comment]
```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-118876982-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-118876982-1');
</script>
```

This needs to be added in. Up for grabs @syedjafer @ragupathis

**Comment by ragupathis** [comment]
Yeah @shrayasr , Will add it. I thought of adding it in the footer of the page for the better page load speed.

Any thoughts on better placement whether to have it in header or footer?

**Comment by shrayasr** [comment]
GA recommends it to be the first child of the <head> tag

**Comment by astronomersiva** [comment]
> GA recommends it to be the first child of the <head> tag

This is because the earlier versions of GA were blocking in nature. The latest versions are not blocking and hence can be placed in the head tag itself. There will be a slight performance hit but this is negligible.

**Comment by astronomersiva** [comment]
Closed in #32
